tests/xharness/Jenkins.cs

```diff
@@ -1989,8 +2019,17 @@ <li>Toggle visibility
 							writer.Write ($"<div class='pdiv {ignoredClass}'>");
 							writer.Write ($"<span id='button_{log_id}' class='expander' onclick='javascript: toggleLogVisibility (\"{log_id}\");'>{defaultExpander}</span>");
 							writer.Write ($"<span id='x{id_counter++}' class='p3 autorefreshable' onclick='javascript: toggleLogVisibility (\"{log_id}\");'>{title} (<span style='color: {GetTestColor (test)}'>{state}</span>) </span>");
-							if (IsServerMode && !test.InProgress && !test.Waiting)
-								writer.Write ($" <span><a class='runall' href='javascript:runtest ({test.ID})'>Run</a></span> ");
+							if (IsServerMode) {
+								writer.Write ($" <span id='x{id_counter++}' class='autorefreshable'>");
+								if (test.Waiting) {
+									writer.Write ($" <a class='runall' href='javascript:stoptest ({test.ID})'>Stop</a> ");
+								} else if (test.InProgress) {
+									// Stopping is not implemented for tasks that are already executing
```

Comment:
Should we add feedback in the xharness web about this fact?

Reply:
We do (indirectly): there's no link to do anything.